docs/guides/social-signin/microsoft.mdx

```diff
@@ -218,6 +213,16 @@ To do that, add the `subject_source` field set to `me` to the social sign-in pro
     - email
 ```
 
+However, some systems use the `id` field returned by the `https://graph.microsoft.com/v1.0/me` endpoint as a subject identifier.
+To make migrating such systems to Kratos easier, you can use the identifier obtained from the `me` endpoint. To do that, update 
+the social sign-in provider config of your Kratos instance with the following.
+
+```yaml
+subject_source: me
+scope:
+  - https://graph.microsoft.com/User.Read
+```
```

Comment:
The code snippet that was used in this document originally was longer to provide context for users. Could you please use the same approach in your PR? 

The original code block also used this cool highlighting feature (`// highlight-start` / `// highlight-end`) - 
https://docusaurus.io/docs/markdown-features/code-blocks#custom-magic-comments
would be great if you used that too :)